docs/production.md

```diff
@@ -1,72 +1,41 @@
 Production Environments
 =======================
 
-This page describes creating and managing production inventory files.
-For details on deploying a production environment, see the [deployment page](/deployment).
+This page describes creating and managing production environments.
 
 ### Directory overview
 
-Production environments live in the `inventories/` folder.
+Environments live in the `environments/` folder.
 
-To add a new production environment you can follow the steps below, replacing `myproject` with your project name.
+To add a new environment you can follow the steps below, replacing `your_environment` with your environment name.
 
 
-Dimagi-managed production environments are maintained in the private `inventories/dimagi/` submodule.
-Dimagi developers should see [the dimagi page](/dimagi) for details on working with this submodule
-in Dimagi-managed environments.
-
-### Initialize Inventory Folder
+### Initialize environment directory
 
 First create a new inventory folder for your environment.
 This is where your project-specific configuration will live. 
 
 ```bash
-cp -r inventories/example inventories/myproject
+cp -r environments/example environments/your_environment
 ```
 ### Update Inventory Files
 
-Edit the `hosts.yml` and `vars.yml` files with your project-specific changes.
+Edit the `inventory` and `vault.yml` files with your project-specific details.
```

Comment:
Could I persuade you to name the inventory file "inventory.ini"? That would be in line with the [filename in commcare-cloud](https://github.com/dimagi/commcare-cloud/blob/master/environments/production/inventory.ini), and the file does use the [INI file format](https://en.wikipedia.org/wiki/INI_file).

```suggestion
Edit the `inventory.ini` and `vault.yml` files with your project-specific details.
```